python/lsst/skymap/packers.py

```diff
@@ -75,11 +78,12 @@ class SkyMapDimensionPacker(DimensionPacker):
     dimensions : `lsst.daf.butler.DimensionGraph`, optional
         The dimensions of data IDs packed by this instance.  Must include
         ``{skymap, tract, patch}``, and may include ``band``.  If not provided,
-        this will be set to include ``band`` if if ``n_bands != 0``.
+        this will be set to include ``band`` if ``n_bands != 0``.
     bands : `~collections.abc.Mapping` [ `str`, `int` ] or `None`, optional
         Mapping from band name to integer to use in the packed ID.  `None` uses
-        a fixed set of bands defined in this class, but this should be avoided
-        in new code.
+        a fixed set of bands defined in this class.  When calling code can
+        enumerate the bands it is likely to see providing an explict mapping is
```

Comment:
"it is likely to see providing"?

Reply:
comma makes all the difference; should be "is likely to see, providing"